app/ai/voice/agents/breeze_buddy/template/approval.py

```
@@ -0,0 +1,215 @@
+    def _scan_entry(entry: Any, source: str) -> None:
+        if not isinstance(entry, dict) or entry.get("approval") is None:
+            return
+        name = entry.get("name") or entry.get("function_name")
+        if not name:
+            logger.warning(
+                f"[approval] Ignoring approval config on a nameless " f"{source} entry"
+            )
+            return
+        try:
+            approval_map[name] = ApprovalConfig.model_validate(entry["approval"])
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)